### DISCUSSION TITLE
How to use @vue/test-utils in Vue playground?

### DISCUSSION THREAD

**Discussion by BlackCrowxyz** [vuejs/test-utils#2206] [General]
I wanted to test my component in a minimal place like playground (and only vue playground !), how can I do that?

**Comment by cexbrayat** [comment]
Hi @BlackCrowxyz 

You can't test your components in the Vue Playground, so I usually use Stacbkblitz when I want something like that

This starter comes with Vue/Vite/Vitest/TypeScript: https://stackblitz.com/github/vuejs/create-vue-templates/tree/main/typescript-vitest?file=src%2Fcomponents%2F__tests__%2FHelloWorld.spec.ts
